tests/flask/test_flask_1.py

```diff
@@ -158,20 +151,20 @@ def terminate():
       flaskSpanAsObject = json.loads(span_list[0].to_json())
       # Check that the expected results are in the flask extended span attributes
       assert flaskSpanAsObject['attributes']['http.method'] == 'GET'
```

Comment:
the test wasn't getting the new span, updates here are largely to actually test `/route2` endpoint